src/object.di

```diff
@@ -679,11 +679,23 @@ bool _xopCmp(in void* ptr, in void* ptr);
 void __ctfeWrite(T...)(auto ref T) {}
 void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
 
+///////////////////////////////////////
+
+template RTInfoImpl(T)
+{
+    immutable RTInfoImpl = __traits(getPointerBitmap,T);
+}
```

Comment:
This would actually get you the deduplication.

```d
template RTInfoImpl(ubyte[] pointers)
{
    immutable data = pointers;
    immutable RTInfoImpl = &data;
}

template RTInfo(T)
{
    immutable RTInfo = RTInfoImpl!(__traits(getPointerBitmap, T));
}
```